libraries/AP_GPS/AP_GPS_NMEA.cpp

```diff
@@ -215,7 +202,7 @@ bool AP_GPS_NMEA::_have_new_message()
         now - _last_GGA_ms > 150) {
         return false;
     }
-    if (_last_VTG_ms != 0 && 
```

Comment:
Unrelated whitespace change

Reply:
@peterbarker san.. 
I will return to the original.